src/EditorFeatures/Core/Implementation/InlineRename/IEditorInlineRenameService.cs

```diff
@@ -224,18 +224,6 @@ internal interface IInlineRenameInfo
         /// locations to rename, as well as any time the rename options are changed by the user.
         /// </summary>
         Task<IInlineRenameLocationSet> FindRenameLocationsAsync(OptionSet optionSet, CancellationToken cancellationToken);
-
-        /// <summary>
-        /// Called before the rename is applied to the specified documents in the workspace.  Return 
-        /// <see langword="true"/> if rename should proceed, or <see langword="false"/> if it should be canceled.
-        /// </summary>
-        bool TryOnBeforeGlobalSymbolRenamed(Workspace workspace, IEnumerable<DocumentId> changedDocumentIDs, string replacementText);
-
-        /// <summary>
-        /// Called after the rename is applied to the specified documents in the workspace.  Return 
-        /// <see langword="true"/> if this operation succeeded, or <see langword="false"/> if it failed.
-        /// </summary>
-        bool TryOnAfterGlobalSymbolRenamed(Workspace workspace, IEnumerable<DocumentId> changedDocumentIDs, string replacementText);
```

Comment:
We've confirmed this isn't used by F# or TypeScript or anything? I don't quite know how they _could_, but it's my job to ask.

Reply:
Not yet :)